addons/crm/res_partner.py

```diff
@@ -12,7 +12,7 @@ def _opportunity_meeting_count(self, cr, uid, ids, field_name, arg, context=None
         # the user may not have access rights for opportunities or meetings
         try:
             for partner in self.browse(cr, uid, ids, context):
-                if partner.is_company:
+                if partner.is_company == 'company':
```

Comment:
Now 'is_company' is a selection field not boolean. (See this commit : https://github.com/odoo/odoo/commit/67447b2fc9eb5436e7ac2b5098373cd0f06b7860)